### ISSUE TITLE
[BottomNavigationView] Wrapping long labels

### ISSUE THREAD

**Issue by pchmielowski** [material-components/material-components-android#1698]
**Is your feature request related to a problem? Please describe.**
I'd like to have an option which makes long labels on `BottomNavigationView` be wrapped instead of ellipsized, like in this example:
![obraz](https://user-images.githubusercontent.com/7473920/92623039-09cb8280-f2c6-11ea-9073-83dc3b3829c5.png)

**Describe the solution you'd like**
Possible solution would be to add new view attributes, such as:
```
app:onLabelOverflow="ellipsize"
app:onLabelOverflow="wrap"
```

**Comment by wcshi** [comment]
The spec explicitly recommends against truncating / ellipsizing text and cautions to avoid wrapping text.

https://material.io/components/bottom-navigation#anatomy

**Comment by pchmielowski** [comment]
@wcshi Yes, exactly - that's why this setting would be useful. I've suggested a possibility to set `app:onLabelOverflow="ellipsize"` because this is a default behaviour right now and it would be nice to keep the backward compatibility.

**Comment by Papashkin** [comment]
The same question for me.
I need similar solution because I need to translate application to other languages, and in some cases, menu label may be ellipsized in the end and look ugly.
